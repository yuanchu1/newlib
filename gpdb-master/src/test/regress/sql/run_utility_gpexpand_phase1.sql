-- start_matchsubs
-- m/^\S+ (.*):.*:.*-(\[\S+\]):-/
-- s/^\S+ (.*):.*:.*-(\[\S+\]):-/TIMESTAMP $1:HOST:USER-$2:-/
-- end_matchsubs

\! echo 'EXPANSION_PREPARE_STARTED:<path> to inputfile' > $COORDINATOR_DATA_DIRECTORY/gpexpand.status
\! gpcheckcat
-- most gppkg actions should be disallowed while gpexpand is in progress
\! gppkg --query no-such-package
-- the only exception is 'build' which has no interaction with the cluster
\! gppkg --build no-such-package
\! rm $COORDINATOR_DATA_DIRECTORY/gpexpand.status
