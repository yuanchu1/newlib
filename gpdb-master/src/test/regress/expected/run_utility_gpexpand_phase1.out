-- start_matchsubs
-- m/^\S+ (.*):.*:.*-(\[\S+\]):-/
-- s/^\S+ (.*):.*:.*-(\[\S+\]):-/TIMESTAMP $1:HOST:USER-$2:-/
-- end_matchsubs
\! echo 'EXPANSION_PREPARE_STARTED:<path> to inputfile' > $COORDINATOR_DATA_DIRECTORY/gpexpand.status
\! gpcheckcat
ERROR: Usage of gpcheckcat is not supported while the cluster is in a reconfiguration state, exit gpcheckcat
-- most gppkg actions should be disallowed while gpexpand is in progress
\! gppkg --query no-such-package
20190220:10:23:39:016760 gppkg:nyu-vm-desktop:u-[INFO]:-Starting gppkg with args: --query no-such-package
20190220:10:23:39:016760 gppkg:nyu-vm-desktop:u-[ERROR]:-gppkg error: ERROR: Usage of gppkg is not supported while the cluster is in a reconfiguration state, exit gppkg
-- the only exception is 'build' which has no interaction with the cluster
\! gppkg --build no-such-package
20190307:15:08:14:017703 gppkg:nyu-vm-desktop:u-[INFO]:-Starting gppkg with args: --build no-such-package
20190307:15:08:14:017703 gppkg:nyu-vm-desktop:u-[INFO]:-Building gppkg
20190307:15:08:14:017703 gppkg:nyu-vm-desktop:u-[ERROR]:-no-such-package is an Invalid directory
20190307:15:08:14:017703 gppkg:nyu-vm-desktop:u-[CRITICAL]:-gppkg failed. (Reason='') exiting...
\! rm $COORDINATOR_DATA_DIRECTORY/gpexpand.status
